src/views/Export/ExportTypePane/index.tsx

```diff
@@ -0,0 +1,279 @@
+                                <Checkbox
+                                    label={_ts('export', 'showEntryGroupsLabel')}
+                                    value={showGroups}
+                                    className={styles.showGroupCheckbox}
+                                    onChange={onShowGroupsChange}
+                                />
+                            )}
+                            {showTextWidgetSelection && (
+                                <TreeSelection
+                                    label={_ts('export', 'textWidgetLabel')}
+                                    value={textWidgets}
+                                    onChange={onTextWidgetsChange}
+                                />
+                            )}
+                            {showContextualWidgetSelection && (
+                                <TreeSelection
+                                    label={_ts('export', 'contextualWidgetLabel')}
+                                    value={contextualWidgets}
+                                    onChange={onContextualWidgetsChange}
+                                />
+                            )}
+                        </div>
+                    </div>
+                )}
+            </div>
+        );
+    }, [
+        contextualWidgets,
+        entryFilterOptions,
+        onContextualWidgetsChange,
+        onReportStructureChange,
+        onReportStructureVariantChange,
+        onShowGroupsChange,
+        onTextWidgetsChange,
+        reportStructure,
+        reportStructureVariant,
+        showGroups,
+        textWidgets,
+    ]);
+
+    const renderExcelOptions = useMemo(() => (
+        <>
+            <Checkbox
+                key="checkbox"
+                label={_ts('export', 'decoupledEntriesLabel')}
+                value={decoupledEntries}
+                onChange={onDecoupledEntriesChange}
+            />
+            <div
+                key="info"
+                className={styles.info}
+            >
+                <Icon
+                    className={styles.icon}
+                    name="info"
+                />
+                <div>
+                    <p>{_ts('export', 'decoupledEntriesTitle2')}</p>
+                    <p>{_ts('export', 'decoupledEntriesTitle')}</p>
+                </div>
+            </div>
+        </>
+    ), [decoupledEntries, onDecoupledEntriesChange]);
+
+    const renderOptions = useMemo(() => {
+        switch (activeExportTypeKey) {
+            case 'word':
+            case 'pdf':
+                return renderWordPdfOptions;
+            case 'excel':
+                return renderExcelOptions;
+            default:
+                return (
+                    <p>
+                        { _ts('export', 'noOptionsAvailable') }
+                    </p>
+                );
+        }
+    }, [renderWordPdfOptions, renderExcelOptions, activeExportTypeKey]);
```

Comment:
Instead of optimizing using useMemo, we should separate them into separate components.